test-server/server.js

```diff
@@ -104,6 +104,11 @@ app.get('/manual-snapshots', (req, res) => {
   res.sendFile(path.join(__dirname, 'fixtures/manual-snapshots.html'));
 });
 
+// Pages fallback
+app.get('/:page', (req, res) => {
+  res.sendFile(path.join(__dirname, `fixtures/pages/${req.params.page}.html`));
```

Comment:
(Not a requested change, just an observation) Oh Codacy 😩 . To be honest I've stopped reading these most of the time because Codacy has so many on each PR. 

Seems the Chromatic repo's Codacy is a lot less noisy. I wonder if there's a setting we can tweak to make this more that way/less spammy.